client/sites-dashboard/components/sites-table-row-loading.tsx

```diff
@@ -0,0 +1,84 @@
+import { LoadingPlaceholder } from '@automattic/components';
+import { css } from '@emotion/css';
+import styled from '@emotion/styled';
+
+type cssSize = number | string;
+
+interface LoadingLogoProps {
+	width: cssSize;
+	height: cssSize;
+}
+
+interface SitesTableRowLoadingProps {
+	columns?: number;
+	logoProps?: LoadingLogoProps;
+	delayMS?: number;
+}
+
+const Row = styled.tr`
+	line-height: 2em;
+	border-bottom: 1px solid #eee;
+`;
+
+const Column = styled.td< { mobileHidden?: boolean } >`
+	padding-top: 12px;
+	padding-bottom: 12px;
+	padding-right: 24px;
+	vertical-align: top;
+`;
+
+const TitleRow = styled.div`
+	display: flex;
+`;
+
+const FullWidth = styled.div`
+	width: 100%;
+`;
+
+const LoadingLogo = styled( LoadingPlaceholder )< LoadingLogoProps >`
+	${ ( { width, height } ) => ( {
+		maxWidth: width,
+		height,
+	} ) }
+	margin-right: 10px;
+`;
+
+const LoadingTitle = styled( LoadingPlaceholder )`
+	max-width: 80%;
+	margin-bottom: 10px;
+`;
+
+const LoadingDomain = styled( LoadingPlaceholder )`
+	max-width: 40%;
+`;
```

Comment:
Let's use CSS objects instead of style declaration soup 😛

Reply:
> Let's use CSS objects instead of style declaration soup 😛

@zaguiini Can you provide an example? @sejas' implementation seems quite common across the codebase. Not sure which alternative pattern you're referring to.

Reply:
```js
const MyStyledComponent = styled.element({
   rule: 'value'
})